### DISCUSSION TITLE
Including Admonitions in Table of Contents

### DISCUSSION THREAD

**Discussion by luca-RLI** [squidfunk/mkdocs-material#4880] [Q&A]
Hi all!

I'm currently working on a documentation project and am in a bit of a bind (let me preface this by saying I'm no expert in markdown and coding generally, but your guides have been extremely helpful even for a newbie such as myself!).

I've noticed collapsible headings are not possible in mkdocs material, so what I did was use admonitions instead. Now, my problem is that using admonitions means that they do not show up under my table of contents as they are not headings. I randomly tried inputs such as:

`####` ??? note "Read more"

`???` ### note "Read more"

`???` note ### "Read more"

Of course, none worked. Is there some kind of feature that allows me to include admonitions in the table of contents (whether as headings or not), or is there just no way I can do this with Material?

Thank you!

**Comment by squidfunk** [comment]
How about:

``` yaml
### Heading

!!! info ""
    ...
```

This will render an admonition without a title, but keep the heading.

**Reply by luca-RLI** [reply]
I did that but unfortunately it doesn't look particularly good:

<img width="533" alt="Image" src="https://user-images.githubusercontent.com/122739572/212972131-eb53a955-3c6b-41b4-a7cb-91951c48db16.png">

**Reply by squidfunk** [reply]
Yes, if you use details (i.e. with `???`) they must have a title. Otherwise there would be nothing to expand them. However, you can use a non-breaking space to fix this:

``` yaml
### Heading

!!! info "&nbsp;"
    ...
```

**Reply by luca-RLI** [reply]
Oh yes, of course! My bad, I should have clarified that what I originally did to keep a heading was exactly that, and then writing "read more" instead of the `"&nbsp;"`. 

The problem is that, as the documentation project I'm working on consists of guidelines, I end up having (say) 20 guidelines in a row with "read more" under each one and it's not the most aesthetically pleasing thing to look at. I would prefer being able to use the guideline itself as an admonition visible in the TOC rather than always adding another "read more".

I appreciate the help!

**Reply by squidfunk** [reply]
Then admonitions might not be the right choice. They are primarily intended for side content, not for main content, which is why it's not adviseable to put headlines into them, especially when they're collapsible.

**Reply by luca-RLI** [reply]
I understand! I needed something to make my page less crowded and long, and since collapsible headings aren't possible, admonitions seemed like the next best thing (I initially thought headings could go in admonitions). I'll probably end up keeping the heading + "read more" admonition combo anyway
